docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 10
+sidebar_label: Get started with Python
+---
+
+# Python Quickstart
+
+Welcome to the Python Quickstart guide for Novu, a powerful notification service that enables you to send multi-channel (SMS, Email, Chat, Push) notifications from your Python applications. In this Quickstart, you'll learn how to seamlessly integrate Novu into your Python project and perform various essential tasks. Let's get started!
```

Comment:
Welcome to Python Quickstart guide for Novu. In this guide, you'll learn how to use Novu in a Python application. Novu is a powerful notification service that enables you to send multi-channel (SMS, Email, Chat, Push) notifications. Let's see how you can seamlessly integrate Novu into your Python project!